static/styles/rendered_markdown.css

```diff
@@ -283,6 +283,25 @@
         margin-right: 0.75em;
     }
 
+    .twitter-attachment {
+        display: block;
+        text-align: right;
+        margin-top: -18px;
+        color: hsl(300, 1%, 38%);
+    }
+
+    .twitter-logo {
+        width: 16px;
+        height: 16px;
+        position: relative;
+        max-width: 100%;
+        display: inline-block;
+    }
+
+    .twitter-timestamp {
+        font-size: 12px;
+    }
```

Comment:
Can you show how this CSS renders tweets that were rendered before this PR?  We'll want to make sure that we don't mangle the formatting of older messages.

Reply:
So a font size change for the date, right?

Reply:
Yes, it's only for the date.